### ISSUE TITLE
misleading documentation

### ISSUE THREAD

**Issue by flying-sheep** [unional/ava-fixture#8]
under **Usage** it says

> // path is the path to the fixture-name folder.

which isn’t true. `path` is `{ casePath: <fixture-name folder> }`

**Comment by unional** [comment]
Thanks. I need to update the Readme.
I just added the baseline functionality.

The option is optional.
```sh
- fixture
  - cases
  - baselines
  - results
```

```ts
fixture(ava, 'fixture/cases') // path = path to the root folder of all fixture cases

fixture(test, 'fixture', { casesPath: 'cases', baselinesPath: 'baselines', resultsPath: 'results' }) // path = path to the relative root of the fixtures.
```

**Comment by unional** [comment]
Now as I think of it, the API is not good. I'll update the API and make another release.

**Comment by unional** [comment]
`0.5` released and updated README. Please see if it is more helpful.

**Comment by unional** [comment]
Released 0.7. It should be ok now. Feel free to continue the discussion.

**Comment by flying-sheep** [comment]
cool!
